src/accessibility/context_menu/ContextMenuButton.tsx

```diff
@@ -0,0 +1,51 @@
+/*
+Copyright 2015, 2016 OpenMarket Ltd
+Copyright 2018 New Vector Ltd
+Copyright 2019 The Matrix.org Foundation C.I.C.
```

Comment:
Are these the right copyright headers here and for all other new files?

Reply:
Well the code is existing just moved and without tracing through exactly which copyright was used when it was created I thought this was safest

Reply:
Ah right I see now, seems fine then.